ci/bin/lint.dart

```diff
@@ -26,6 +26,10 @@ more information on addressing these issues please see:
 https://github.com/flutter/flutter/wiki/Engine-Clang-Tidy-Linter
 ''';
 
+const List<String> clangTidyDefineArgs = <String>[
```

Comment:
I thought we were using the `compile_commands.json` file to find the arguments specified to each translation unit. If not, we should wire that up instead as adding each argument here based on changes to the GN rules will get unmaintainable.